### ISSUE TITLE
Add Close button to mod browser

### ISSUE THREAD

**Issue by UltraBlackLinux** [ATLauncher/ATLauncher#638]
**Is your feature request related to a problem? Please describe.**
I am running wayfire and I disabled the xwayland window buttons since I just want those clean buttonless window borders. For some reason I am unable to close the mod browser using the default close window key-combination.

**Describe the solution you'd like**
Either get the window to accept being closed externally or add a close button

**Comment by thiccchris** [comment]
1. Since this is an old Issue, are you still having this problem? (and or can I get a setup or something similar to replicate and test)

2. Is it only the browse mods popup or other pop ups like: Export popup, add mod, etc...?

**Comment by UltraBlackLinux** [comment]
good point, I'm no longer on wayfire and neither have I played minecraft for like a year now so yeah, I guess I'll close this
